src/writer.rs

```diff
@@ -363,6 +376,19 @@ impl<'a, W> ElementWriter<'a, W> {
         self.start_tag.extend_attributes(attributes);
         self
     }
+
+    /// Push a new line inside an element.
+    pub fn new_line(mut self) -> Self  {
+        if let Some(i) = &self.writer.indent {
+            let state = self.start_tag.state;
+            match state {
+                AttributeIndent::NoneAttributesWritten => self.start_tag.state = AttributeIndent::Indent(i.current_indent_len),
+                AttributeIndent::SomeAttributesWritten(isize) => self.start_tag.state = AttributeIndent::Indent(isize),
+                AttributeIndent::Indent(_) => {self.start_tag.push_newline(i.current())},
```

Comment:
You need to indent on `Indent(_) + i.current()`, pseudocode:
```suggestion
                AttributeIndent::Indent(ind) => self.start_tag.push_newline(ind + i.current()),
```